docs/formats/zst.md

```diff
@@ -365,7 +365,7 @@ column values from.
 The top-level reassembly fetches column values as a `<record_column>`.
 
 For any `<record_column>`, a value from each field is read from each field's column,
-accounting for the presence column indicating nil,
+accounting for the presence column indicating null,
```

Comment:
The word `nil` never appears anywhere else in this doc and `null` is referenced several times when speaking about the role of the presence column, so I assume that's what's intended here.